app/views/relationships/show.html.haml

```diff
@@ -8,8 +8,8 @@
   .filter-subset
     %strong= t 'relationships.relationship'
     %ul
-      %li= filter_link_to t('accounts.following', count: current_account.following_count), relationship: nil
-      %li= filter_link_to t('accounts.followers', count: current_account.followers_count), relationship: 'followed_by'
```

Comment:
Maybe we could just access `accounts.followers.other` directly? Just thinking about reducing translation overhead

Reply:
Unfortunately no, they are similar in English, but may differ in other languages (they do in Russian actually). So I think it's the best solution for us to have separate strings based on context, that's what issue #10863 was exactly in the first place.

---

Also, `accounts.following` seems to be incorrect, because in original context it is used as plural, but in translation file it is a string:

```haml
-# .../accounts/_header.html.haml
%span.counter-number= number_to_human account.following_count, strip_insignificant_zeros: true
%span.counter-label= t('accounts.following', count: account.following_count)
```

```yml
# en.yml
    followers:
      one: Follower
      other: Followers
    following: Following
#   ^^^^^^^^^
```

Want me to fix that in this or another PR?